### PR TITLE
Translating getting-started.md

### PR DESCRIPTION
**Pull request by mhmdAljefri**
…nslations #1

### CODE REVIEW

**Comment by mhmdAljefri** [comment]
I think I'm done here if any issue ping please. React help my in many application this time to do the same @iRayan7

**Comment by mhmdAljefri** [comment]
Wa Alikum Salam, Sure I will. Thank you

File: content/docs/getting-started.md [hunk 1]
```diff
@@ -18,109 +18,111 @@ redirect_from:
...
-* If you prefer to **learn concepts step by step**, start with our [guide to main concepts](/docs/hello-world.html).
+مثل أي تقنية غير مألوفة لك، تملك React مسارًا تعليميًّا مخصصًّا بها. بقليل من الممارسة والصبر، *ستتقن* React وتحصل على ما تريده بالتأكيد.
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
 
-### First Examples {#first-examples}
+### الأمثلة الأولى {#first-examples}
 
-The [React homepage](/) contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.
+[الصفحة الرئيسية لتوثيق React](/)بضعة أمثلة بسيطة عن React. حتى إن تعرف أي شيء عن React بعد، جرب تشغيل الأمثلة ثم تعديلها مع ملاحظة التغيير الذي سيحصل للمخرجات.
```
Comment (iRayan7) [inline content/docs/getting-started.md]: space before 'بضعة'

**Comment by iRayan7** [comment]
Merged! 🎉
Congrats on your first amazing contribution, Muhammad! Keep up it up 💪

**Comment by mhmdAljefri** [comment]
lol, thanks bro It's really wonderful  experiment... Will keep it up :tada:

**Pull request merged**